### ISSUE TITLE
DataLossError when trying to open pre-trained checkpoints! 

### ISSUE THREAD

**Issue by cbsudux** [amirbar/speech2gesture#6]
@amirbar I'm facing an error when I run `python -m audio_to_multiple_pose_gan.predict_audio --audio_path oliver_test.wav -output_path tmp_output/ --checkpoint Gestures/pretrained_models/oliver/ckpt-step-296700.ckp.data-00000-of-00001 --speaker oliver -ag audio_to_pose_gans --gans 1`  for inference on audio.

This is the output error I get:
`DataLossError (see above for traceback): Unable to open table file Gestures/pretrained_models/conan/ckpt-step-296700.ckp.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]`

Any idea what's causing this? (Do let me know if you want the full log)

**Comment by amirbar** [comment]
yes, please try pass in the checkpoint param with as follows:
`Gestures/pretrained_models/oliver/ckpt-step-296700.ckp`
